Makefile

```diff
@@ -57,9 +60,13 @@ docker: $(DOCKERS)
 docker_device_opcua_go:
 	docker build \
 		--label "git_sha=$(GIT_SHA)" \
-		-t edgexfoundry/device-opcua-go:$(GIT_SHA) \
-		-t edgexfoundry/device-opcua-go:$(VERSION)-dev \
+		--build-arg ADD_BUILD_TAGS=$(ADD_BUILD_TAGS) \
+		-t edgexfoundry/device-opcua-goa:$(GIT_SHA) \
+		-t edgexfoundry/device-opcua-goa:$(VERSION)-dev \
```

Comment:
Did you mean to change this to `-goa`?

Reply:
Whoops, I didn't want to overwrite the one I had locally and forgot to change it back. Thanks.